configs/milvus.yaml

```diff
@@ -65,7 +65,8 @@ mysql:
 
 # please adjust in embedded Milvus: /tmp/milvus/data/
 localStorage:
-  path: /var/lib/milvus/data/
+  path: /var/lib/milvus/data/ # used to storage rdb datas, which must be persistent
+  tempFilePath: /milvus/data/ # used for build disk index and load disk index or raw data, data can be cleaned when restart.
```

Comment:
why we need a actual config?
doesn't make sense to me to have different configs.
There should be only one config localStorage.path